src/routes/PersonalPages/index.tsx

```diff
@@ -176,6 +192,16 @@ const Competences = ({ routes }: RoutesNavigationListProps) => {
         simul accusata no ius. Volumus corpora per te, pri lucilius salutatus iracundia ut. Mutat posse voluptua quo cu,
         in albucius nominavi principes eum, quem facilisi cotidieque mel no.
       </p>
+      <h2 className="mb-6 text-heading-3">Osaamiseni lähteiden mukaan</h2>
+      <Accordion title={<span className="truncate text-heading-5 text-secondary-gray">Työpaikkojen mukaan</span>}>
+        <span className="mb-5 mt-3 flex border border-b-2 border-inactive-gray"></span>
```

Comment:
Used the existing Accordion component for now.
- The toggle button icon is not same
- Component did not have the "line" itself, so roughly created the line here (obviously it will get hidden in this way when closed).

So, should we move the Accordion as a component to design system and add some configurability to add the differences or which way tou think would be good in this case?

Reply:
Accordion definitely sounds like it should be part of design system, but currently I don't have a strong opinion, as I'm not aware of all the places where it will be used. Will the design system be used in "ohjaaja" portion of the application? If so, it'll make even more sense to have the accordion in design system.

Reply:
Okay, I will move the Accordion component to design system side in separate PR.